snmp/datadog_checks/snmp/data/default_profiles/extreme-switching.yaml

```diff
@@ -0,0 +1,123 @@
+extends:
+  - _base.yaml
+  - _generic-if.yaml
+  - _generic-entity-sensor.yaml
+
+sysobjectid:
+  - 1.3.6.1.4.1.1916.2.89
+  - 1.3.6.1.4.1.1916.2.343
+
+metadata:
+  device:
+    fields:
+      vendor:
+        value: "extreme-networks"
```

Comment:
Ideally, we should match the metadata vendor with the profile name vendor prefix for consistency.

Reply:
when I researched this I found it's extreme-networks and not just extreme (similar to juniper devices the vendor is juniper-networks and not just juniper)

Reply:
I'm just a bit worried that this might increase the likelihood of multiple vendor name values, for different profile of the same vendor.

Reply:
for this case we only have this profile and for the future when adding new profiles we'll just have to match the value that we have here
I'm inclined to keep this because it's the official vendor name and also in the MIB this is how the enterprise is defined:
https://www.circitor.fr/Mibs/Html/E/EXTREME-BASE-MIB.php#extremenetworks